source/_less/position.less

```diff
@@ -0,0 +1,144 @@
+@import 'vars';
```

Comment:
Please rename the file to denote that it is intended to be used as a mixin. Maybe `mix-position.less`